src/styles/core/_breakpoints.scss

```diff
@@ -332,11 +332,17 @@ $breakpoint-attributes: (
     padding: 0 20px;
     box-sizing: border-box;
 
-    // make sure there is 120px space around the content on xlarge
-    @include breakpoints-from(xlarge) {
+    // make sure there is 120px space between the content and nav bar on xl and l
+    @include breakpoints-from(large) {
```

Comment:
This makes the Large breakpoint content pretty crammed, if the sidebar is opened to the widest position. Is that intended?